config_management/group_vars/all

```diff
@@ -4,7 +4,7 @@ terraform_version: 0.8.5
 docker_version: 1.11.2
 kubernetes_version: 1.5.2
 kubernetes_cni_version: 0.3.0.1
-kubernetes_token: 123456.0123456789123456
+kubernetes_token: '123456.0123456789123456'
```

Comment:
haha, I just realised how this broke and how it just-about-worked before. Basically it always got interpreted as a float, but the few-to-many numbers after the decimal point got dropped and `kubeadm` used to be happy, but now it wants to see a key of certain length.